frontend/src/pages/units/Unit.tsx

```diff
@@ -190,7 +190,7 @@ function render(props: Props, state: State) {
                 </Button>
             </th>
             <td>{devices}</td>
-            <td>{ props.properties.find(e => e.id === props.propertyId )?.name }</td>
+            <td style={{padding:19.5}}>{ props.properties.find(e => e.id === props.propertyId )?.name }</td>
```

Comment:
I'm confused on why we'd need to apply some custom styling. I'd think that the standard bootstrap classes would do what we want 🤔 (we might be using them wrong)